src/mock-data.json

```diff
@@ -233,7 +233,7 @@
       "field_syndicated_content": "True",
       "field_modified_date":"2019-01-22T16:00:55-05:00",
       "field_published_date":"2019-01-09T11:20:57-05:00",
-      "field_tl_": "268:::269:::270:::272:::273",
+      "field_tl_": "268:::269:::270:::272:::273:::1118",
```

Comment:
@runrog we need to remove `field_syndicated_content` and `field_featured_podcast` from these mocks and update any references to them in the code as well, as those fields will get removed from production after this is deployed.

Reply:
`field_syndicated_content` is in a lot of places including templates/tests FYI. going to take some rebuilding here.